iot-hub/Samples/service/JobsSample/Parameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using CommandLine;
+
+namespace JobsSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "HubConnectionString",
```

Comment:
Did we decide to go with `c` everywhere for connection string? If not, then we stick with `p`.

Reply:
Do we want `c` to be specific for a Hub Connection string and `p` to be specific for a device connection string? That would make sense to me but everything could also just be `p` as well